config/scheduler-chromeos.yaml

```diff
@@ -389,3 +389,53 @@ scheduler:
     <<: *test-job-arm64-mediatek
     platforms:
       - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-av1-chromium-10bit
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - mt8195-cherry-tomato-r2
+
+  - job: v4l2-decoder-conformance-h264
+    <<: *test-job-arm64-mediatek
+    platforms:
+      - sc7180-trogdor-kingoftown
+      - sc7180-trogdor-lazor-limozeen
```

Comment:
This won't work as `test-job-arm64-mediatek` builds the kernel for mediatek devices only; please create additional scheduler entries for qualcomm devices.

Reply:
I'll update it in next change.